apps/src/sites/studio/pages/signup.js

```diff
@@ -136,10 +147,10 @@ window.SignupManager = function (options) {
   function logAnalyticsEvent(eventName, extraData = {}) {
     const streamName = "analysis-events";
     const study = "signup_school_dropdown";
-    const studyGroup = shouldShowSchoolDropdown() ? "show_school_dropdown" : "control";
+    const studyGroup = getSchoolDropdownStudyGroup();
 
     let dataJson = extraData;
-    if (!!window.optimizely && !!window.optimizely.data) {
+    if (window.optimizely && window.optimizely.data) {
```

Comment:
The !! was unnecessary, as these are normal objects if present, which will never be falsy